apollo/integrations/db/postgres_proxy_client.py

```diff
@@ -0,0 +1,45 @@
+from typing import Dict, Optional
+
+import psycopg2
+from psycopg2 import DatabaseError
+from psycopg2.errors import QueryCanceled, InsufficientPrivilege  # noqa
+
+from apollo.integrations.db.base_db_proxy_client import BaseDbProxyClient
+
+_ATTR_CONNECT_ARGS = "connect_args"
+
+
+class PostgresProxyClient(BaseDbProxyClient):
```

Comment:
Just wondering, should we have this class extending the Redshift one? or maybe the other way around? It looks the code is almost the same

Reply:
I think redshift extending postgres makes sense as Redshift uses Postgres under the hood